packages/@cdktf/hcl2cdk/test/expressionToTs.test.ts

```diff
@@ -148,7 +200,11 @@ describe("expressionToTs", () => {
   test("convert a local reference", async () => {
     const expression = `"simple-\${local.foo}"`;
     const scope = getScope({ locals: ["foo"] });
-    const result = await convertTerraformExpressionToTs(expression, scope, "");
+    const result = await convertTerraformExpressionToTs(
+      expression,
+      scope,
+      getType
+    );
     expect(code(result)).toMatchInlineSnapshot(
       `"\\"simple-\${\\" + foo + \\"}\\""`
```

Comment:
given that this expects `foo` to be the variable name for that local – wouldn't this collide in the case of HCL having both a `var.foo` and a `local.foo`?
Or am I missing something?

Reply:
(this is probably not related to this PR, but rather an existing shortcoming – if true)